dev-build/automake/automake-1.17.recipe

```diff
@@ -68,7 +68,7 @@ TEST_REQUIRES="
 BUILD()
 {
 	./bootstrap
-	PERL="/bin/env perl" runConfigure ./configure
+	PERL="$(which perl)" runConfigure ./configure
```

Comment:
Why do we need to specify this anyway? Does it work without explicitly specifying perl?

This was apparently introduced in f0a5b24ee4d93a44db975965ec378f6359c8e0bf in 2013 but without an explanation.